apis/v1alpha1/baselineadminnetworkpolicy_types.go

```diff
@@ -129,17 +161,24 @@ type BaselineAdminNetworkPolicyEgressRule struct {
 	// Currently the following actions are supported:
 	// Allow: allows the selected traffic
 	// Deny: denies the selected traffic
+	//
+	// Support: Core
+	//
 	Action BaselineAdminNetworkPolicyRuleAction `json:"action"`
 
 	// To is the list of destinations whose traffic this rule applies to.
-	// If any AdminNetworkPolicyPeer matches the destination of outgoing
+	// If any AdminNetworkPolicyEgressPeer matches the destination of outgoing
 	// traffic then the specified action is applied.
 	// This field must be defined and contain at least one item.
 	// +kubebuilder:validation:MinItems=1
-	To []AdminNetworkPolicyPeer `json:"to"`
+	// +kubebuilder:validation:MaxItems=100
+	//
+	// Support: Core
+	//
+	To []AdminNetworkPolicyEgressPeer `json:"to"`
```

Comment:
why now the differentiation between Ingress and Egress?

Reply:
The network selector was added to egress (egress controls by CIDR block). We didn't include Ingress yet due to the difficulties reliably determining the source IP of ingress traffic.

https://network-policy-api.sigs.k8s.io/npeps/npep-126-egress-traffic-control/#non-goals